Content.Server/StationEvents/Events/KudzuGrowth.cs

```diff
@@ -0,0 +1,56 @@
+using Content.Shared.Station;
+using Robust.Shared.GameObjects;
+using Robust.Shared.IoC;
+using Robust.Shared.Log;
+using Robust.Shared.Map;
+using Robust.Shared.Maths;
+using Robust.Shared.Random;
+
+namespace Content.Server.StationEvents.Events;
+
+public class KudzuGrowth : StationEvent
+{
+    [Dependency] private IRobustRandom _robustRandom = default!;
+    [Dependency] private IEntityManager _entityManager = default!;
+
+    public override string Name => "KudzuGrowth";
+
+    public override string? StartAnnouncement =>
+        "Attention crew, we have detected a Type 2 Biological Invader on-station, that poses potentially serious threat to crew productivity. We advise you to exterminate it.";
+
+    public override string? StartAudio => "/Audio/Announcements/bloblarm.ogg";
+
+    public override int EarliestStart => 15;
+
+    public override int MinimumPlayers => 15;
+
+    // Get players to scatter a bit looking for it.
+    protected override float StartAfter => 50f;
+
+    // Give crew at least 9 minutes to either have it gone, or to suffer another event. Kudzu is not actually required to be dead for another event to roll.
+    protected override float EndAfter => 60*4;
```

Comment:
this is 4 minutes

Reply:
I'm counting the minimum time between events in that. 4 + 5.